src/controllers/handlers/about-handler.ts

```diff
@@ -3,49 +3,71 @@ import { HandlerContextWithPath } from '../../types'
 export type About = {
   healthy: boolean
   configurations: {
-    commsProtocol: string
+    realmName?: string
   }
   content: {
     healthy: boolean
+    version?: string
+    commitHash?: string
   }
   comms: {
+    protocol: string
     healthy: boolean
+    version?: string
+    commitHash?: string
   }
   lambdas: {
     healthy: boolean
+    version?: string
+    commitHash?: string
+  }
+  bff: {
+    healthy: boolean
+    commitHash?: string
+    userCount: number
   }
 }
 
 // handlers arguments only type what they need, to make unit testing easier
 export async function aboutHandler(
   context: Pick<
-    HandlerContextWithPath<'serviceDiscovery' | 'logs' | 'metrics' | 'config' | 'fetch', '/explorer-configuration'>,
+    HandlerContextWithPath<
+      'serviceDiscovery' | 'status' | 'realm' | 'logs' | 'config' | 'fetch' | 'rpcSessions',
+      '/about'
+    >,
     'url' | 'components'
   >
 ) {
-  const { logs, config, fetch, serviceDiscovery } = context.components
+  const { logs, realm, config, fetch, status, serviceDiscovery, rpcSessions } = context.components
 
-  const logger = logs.getLogger('explorer-configuration')
+  const logger = logs.getLogger('about')
   const commsProtocol = await config.requireString('COMMS_PROTOCOL')
-  const lambdasUrl = await config.requireString('LAMBDAS_URL')
 
-  const body: About = {
+  const userCount = rpcSessions.sessions.size
+  const result: About = {
     healthy: false,
-    configurations: {
-      commsProtocol
-    },
     content: {
       healthy: false
     },
+    configurations: {
+      realmName: undefined
+    },
     comms: {
+      protocol: commsProtocol,
       healthy: false
     },
     lambdas: {
       healthy: false
+    },
+    bff: {
+      healthy: true,
+      commitHash: await config.getString('COMMIT_HASH'),
+      userCount
     }
   }
 
   try {
+    const lambdasUrl = await config.requireString('LAMBDAS_URL')
```

Comment:
can we parse it to an URL, so if someone configures it with a leading `/` then it will work anyway?

Reply:
done